.env

```diff
@@ -0,0 +1,5 @@
+REACT_APP_IMAGE_URL="https://apps.perseids.org/beta-code/favicon-128x128.png"
+REACT_APP_SITE_NAME="Beta Code Converter"
```

Comment:
`REACT_APP_SITE_NAME` should be `The Perseids Project`. According to the Open Graph [specification](http://ogp.me/):

> og:site_name - If your object is part of a larger web site, the name which should be displayed for the overall site. e.g., "IMDb".